package.json

```diff
@@ -100,6 +100,7 @@
     "@types/vscode": "^1.0.0",
     "@vscode/test-electron": "^2.0.0",
     "esbuild": "^0.14.0",
+    "ovsx": "^0.5.0",
```

Comment:
Is this needed? I don’t see it being used?

Reply:
It’s used here: https://github.com/remarkjs/vscode-remark/pull/112/files#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R20

Reply:
Ah OK due to npx I thought it was being pulled in then. This is better